docker/local/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.18-alpine AS builder
```

Comment:
Not a big deal, but the official supported go version for Polygon Edge is `1.17`.

Reply:
Topic for another PR but we need to consider upgrading to 1.18